client/gui/SDL_Extensions.cpp

```diff
@@ -361,6 +361,74 @@ void CSDL_Ext::update(SDL_Surface * what)
 	if(0 !=SDL_UpdateTexture(screenTexture, nullptr, what->pixels, what->pitch))
 		logGlobal->error("%s SDL_UpdateTexture %s", __FUNCTION__, SDL_GetError());
 }
+
+uint8_t lerp(uint8_t a, uint8_t b, float f)
+{
+	return a + std::round((b-a)*f);

```

Comment:
Please add spaces around binary operations 
```suggestion
	return a + std::round((b - a) * f);
```